portfolio/src/main/webapp/statistics.html

```diff
@@ -23,7 +24,7 @@
     <link href="css/styles.css" rel="stylesheet" />
     <link href="css/custom.css" rel="stylesheet" />
   </head>
-  <body id="page-top" onload="statisticsOnLoad()">
+  <body id="page-top" onload="statisticsOnLoad(); chartsOnLoad();">
```

Comment:
you should add drawTimeSeriesChart to the statisticsOnLoad() function so that we aren't calling 2 functions in the onload call.

Reply:
Just added to statisticsOnLoad(). If we end up splitting the statistics page into 2, I'll go back to having a chartsOnLoad() function.